easybuild/easyconfigs/c/Caffe/Caffe-1.0-intel-2017b-CUDA-9.1.85-Python-2.7.14.eb

```diff
@@ -0,0 +1,68 @@
+# -*- mode: python; -*-
+# EasyBuild reciPY for Caffe as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2017 UL HPC
+# Authors::   UL HPC Team <hpc-sysadmins@uni.lu>
+# License::   MIT/GPL
+#
+easyblock = 'CMakeMake'
+
+name = 'Caffe'
+version = '1.0'
+
+versionsuffix = '-Python-%(pyver)s'
+cudaversion = '9.1.85'
+versionsuffix = '-CUDA-%s-Python-%%(pyver)s' % cudaversion
```

Comment:
Hmm, I'm not sure if we really want the `CUDA` version in the `versionsuffix`, since that makes this quite long...

Maybe just add a `-gpu` instead?

Reply:
well, we also have Caffe-rc3-foss-2016a-CUDA-7.5.18-Python-2.7.11.eb ...

Reply:
and we also have different CUDA versions ...

Reply:
@boegel

Reply:
OK, then leave as is, it indeed gives flexibility w.r.t. CUDA version.